tests/test_coordinate_extraction.py

```diff
@@ -228,3 +228,29 @@ def test_CoordinateExtractor_get_coordinates_from_lines_rect():  # noqa: D103
     coordinates = extractor.get_coordinates_from_lines(lines, page=1)
     assert coordinates[0].east.coordinate_value == 2695782
     assert coordinates[0].north.coordinate_value == 1260032
+
+
+def test_get_signle_decimal_coordinates():
+    """Test the extraction of decimal coordinates from a list of text lines."""
+    lines = _create_simple_lines(["start", "615.790.6 / 157.500.5", "end"])
```

Comment:
Maybe the "start" and "end" don't need to be included in every test? It's good to test that the coordinates can be found within other text lines, but I would only test this once (in `test_CoordinateExtractor_get_coordinates_from_lines_rect`). Testing that the extraction of decimal values works, should ideally test just that, and don't mix in other redundant context such as these "start" and "end" lines.

Reply:
Understood! Thank you!